apps/estuary/lib/estuary_web/live/live_view_helper.ex

```diff
@@ -0,0 +1,23 @@
+defmodule EstuaryWeb.LiveViewHelper do
+  def filter_events(events, nil), do: events
+
+  def filter_events(events, ""), do: events
+
+  def filter_events(events, value) do
+    Enum.filter(events, fn event ->
+      search_contains?(event["author"], value) ||
+        search_contains?(event["create_ts"], value) ||
+        search_contains?(event["data"], value) ||
+        search_contains?(event["type"], value)
```

Comment:
Do all of these need to be true, or just one of them?

Reply:
any one is fine